third_party/amd/backend/compiler.py

```diff
@@ -94,6 +94,10 @@ def path_to_rocm_lld():
         lld = Path("/usr/bin/ld.lld")
         if lld.is_file():
             return lld
+        if "ROCM_PATH" in os.environ:
```

Comment:
Actually will an environment variable like `TRITON_HIP_LLD_PATH` solving your problem? If we are relying on environment variables, I'd rather to have it being precise and prevail all other options, like what we do for `libamdhip64.so` with `TRITON_LIBHIP_PATH`.